geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped goal
float32 tolerance
---
float32 length
